packages/sygma-ui/src/App.tsx

```diff
@@ -44,9 +45,14 @@ const AppWrap: React.FC<{
 
   const setConfig = async () => {
     if (!window.__RUNTIME_CONFIG__) {
-      const config = await getSygmaConfig();
-      if (config.error) {
-        setErrMessage(config.error.message ?? config.error.name);
+      // const config = await getSygmaConfig();
+      // const sharedConfig = await getSharedConfig()
+      const config = await configMerger();
+      if ((config as ConfigError).error) {
+        setErrMessage(
+          (config as ConfigError).error.message ??
+            (config as ConfigError).error.name
```

Comment:
hey @enemycnt why are we using here property called `name`? In the config function if there is an error it returns an error with a message but not a name, so here I'm a little bit confused

Reply:
That is to catch connection error and server error from config server in a same time